modules/reference/pages/systems/system-details/sd-notes.adoc

```diff
@@ -1,15 +1,11 @@
 [[sd-notes]]
-= SD Notes
+= Notes
 
-This subtab provides a place to create notes about the system.
+The menu:Systems[Details > Notes] section lists notes about the selected client.
```

Comment:
The menu "Systems > Details > Notes" does not exist.  See the other PR I just commented.

Reply:
And see my other reply ;)